server/routes/plants.js

```diff
@@ -1,25 +1,19 @@
-const { createPlant, getPlants } = require('../controllers/plantController')
+const { createPlant, getPlants, getPlant, editPlant, deletePlant } = require('../controllers/plantController')
 const express = require('express')
 const router = express.Router()
 
 // GET all plants
 router.get('/', getPlants)
 
 // GET a single plant
-router.get('/:id', (req, res)=> {
-  res.json({message: `GET one plant with id ${req.params.id}`})
-})
+router.get('/:id', getPlant)
 
 // POST a single plant (create)
-router.post('/', createPlant)
+router.post('/new-plant', createPlant)
 
 // DELETE a single plant
-router.delete('/:id', (req, res) => {
-  res.json({message: 'DELETE one plant'})
-})
+router.delete('/:id', deletePlant)
```

Comment:
Nicely done with these - as you have more functionality it pays off to have things tucked away in their own files